src/main/scala/no/ndla/conceptapi/service/ConverterService.scala

```diff
@@ -144,6 +141,17 @@ trait ConverterService {
         case Right(None)            => toMergeInto.articleId
       }
 
+      val newMetaImage = updateConcept.metaImage match {
+        case Left(_)     => Seq.empty
+        case Right(None) => toMergeInto.metaImage
+        case Right(Some(_)) => {
+          val domainMetaImage = updateConcept.metaImage
+            .map(m => domain.ConceptMetaImage(m.get.id, m.get.alt, updateConcept.language))
+            .toSeq
+          mergeLanguageFields(toMergeInto.metaImage, domainMetaImage)
+        }
+      }
```

Comment:
Det her er vel feil. Det her vil slette metaImage for alle språk dersom man sender inn `Left(_)`?

Unngå å bruke `.get` om mulig, det kan kaste exceptions om optionen er `None`.
Vi kan også la `mergeLanguageFields` håndtere manglende felt slik som tidligere.


```suggestion
      val newMetaImage = updateConcept.metaImage match {
        case Left(_) => toMergeInto.metaImage.filterNot(_.language == updateConcept.language)
        case Right(meta) =>
          val domainMetaImage = meta
            .map(m => domain.ConceptMetaImage(m.id, m.alt, updateConcept.language))
            .toSeq
          mergeLanguageFields(toMergeInto.metaImage, domainMetaImage)
      }
```